examples/hacker_news_assets/hacker_news_assets/repository.py

```diff
@@ -0,0 +1,77 @@
+import os
```

Comment:
If an organization had 100 asset groups and 1.5 jobs per asset group, how would we recommend they structure things?  Asking because the HN example is meant to demonstrate how to structure large projects, and, even we don't fully need it for our three asset groups, I still think it's worth using that structure.

Reply:
If we have a file like that, I think we should call it jobs.py instead of schedules.py, because not every job will have a schedule, but every schedule will have a job.

Reply:
how's this looking? https://github.com/dagster-io/dagster/tree/yuhan/3.14-hacker_news_assets/examples/hacker_news_assets/hacker_news_assets